cluster-scope/overlays/nerc-ocp-test/kustomization.yaml

```diff
@@ -55,7 +55,7 @@ configMapGenerator:
   namespace: openshift-config
   behavior: merge
   literals:
-    - ack-4.15-kube-1.29-api-removals-in-4.16=true
```

Comment:
do we know why this was .16 instead of .18?

Reply:
@joachimweyl I don't think there were any api removals between 4.15 -> 4.18 so we didn't have to update it for those